charts/kubernikus-monitoring/templates/_prometheus.yaml.tpl

```diff
@@ -215,48 +110,76 @@ scrape_configs:
     action: replace
     replacement: ${1}
 
-- job_name: 'kube-system/kubelet'
+- job_name: kube-system/kubelet
+  scheme: https
   kubernetes_sd_configs:
   - role: node
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  tls_config:
+    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    insecure_skip_verify: true
   relabel_configs:
-  - action: labelmap
+  - separator: ;
     regex: __meta_kubernetes_node_label_(.+)
-  - target_label: component
+    replacement: $1
+    action: labelmap
+  - separator: ;
+    regex: (.*)
+    target_label: component
     replacement: kubelet
-  - action: replace
-    source_labels: [__meta_kubernetes_node_name]
-    target_label: instance
-  - source_labels: [__address__]
     action: replace
-    target_label: __address__
-    regex: ([^:;]+):(\d+)
-    replacement: ${1}:10255
-  - source_labels: [__scheme__]
+  - source_labels: [__meta_kubernetes_node_name]
+    separator: ;
+    regex: (.*)
+    target_label: instance
+    replacement: $1
     action: replace
-    target_label: __scheme__
-    regex: https
-    replacement: http
+  metric_relabel_configs:
+    - source_labels: [ id ]
+      action: replace
+      regex: ^/system\.slice/(.+)\.service$
+      target_label: systemd_service_name
+      replacement: '${1}'
+    - source_labels: [ id ]
+      action: replace
+      regex: ^/system\.slice/(.+)\.service$
+      target_label: container_name
+      replacement: '${1}'
 
 - job_name: 'kubernetes-cadvisors'
   scheme: https
+  metrics_path: /metrics/cadvisor
   tls_config:
     ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    insecure_skip_verify: true
   bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
   kubernetes_sd_configs:
     - role: node
   relabel_configs:
     - action: labelmap
       regex: __meta_kubernetes_node_label_(.+)
-    - target_label: __address__
-      replacement: kubernetes.default:443
-    - source_labels: [__meta_kubernetes_node_name]
-      regex: (.+)
-      target_label: __metrics_path__
-      replacement: /api/v1/nodes/${1}:4194/proxy/metrics
-
-# Static Targets 
-#
-- job_name: 'kubernikus-prometheus'
-  metrics_path: /metrics
-  static_configs:
-    - targets: ['localhost:9090']
+  metric_relabel_configs:
+    - source_labels: [ id ]
+      action: replace
+      regex: ^/system\.slice/(.+)\.service$
+      target_label: systemd_service_name
+      replacement: '${1}'
+    - source_labels: [ id ]
+      action: replace
+      regex: ^/system\.slice/(.+)\.service$
+      target_label: container_name
+      replacement: '${1}'
+    - source_labels: [ namespace ]
```

Comment:
Is this correct? That we drop everthing not running in `Kube-system`?

Reply:
Nope. Thanks for spotting this!